packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/DepositStatus/index.tsx

```diff
@@ -34,7 +34,8 @@ const DepositStatus = (props) => {
     Success: (val) =>
       props.formValues?.order?.state === 'CLEARED' ||
       props.formValues?.order?.state === 'COMPLETE' ||
-      props.formValues?.order?.state === 'COMPLETED' ? (
+      props.formValues?.order?.state === 'COMPLETED' ||
+      props.formValues?.order?.state === 'MANUAL_REVIEW' ? (
```

Comment:
and prevent hard code strings duplication

Reply:
actually there are 2 diff COMPLETE and COMPLETED :)